diff --git a/maven-shade-plugin/src/main/java/org/apache/maven/plugins/shade/resource/ManifestResourceTransformer.java b/maven-shade-plugin/src/main/java/org/apache/maven/plugins/shade/resource/ManifestResourceTransformer.java
index 644fad6..74bf3b7 100644
--- a/maven-shade-plugin/src/main/java/org/apache/maven/plugins/shade/resource/ManifestResourceTransformer.java
+++ b/maven-shade-plugin/src/main/java/org/apache/maven/plugins/shade/resource/ManifestResourceTransformer.java
@@ -47,7 +47,7 @@ public class ManifestResourceTransformer
     // Configuration
     private String mainClass;
 
-    private Map<String, Attributes> manifestEntries;
+    private Map<String, String> manifestEntries;
 
     // Fields
     private boolean manifestDiscovered;
@@ -101,7 +101,7 @@ public class ManifestResourceTransformer
 
         if ( manifestEntries != null )
         {
-            for ( Map.Entry<String, Attributes> entry : manifestEntries.entrySet() )
+            for ( Map.Entry<String, String> entry : manifestEntries.entrySet() )
             {
                 attributes.put( new Attributes.Name( entry.getKey() ), entry.getValue() );
             }
